source/Assets/Plugins/Didomi/Scripts/Didomi.cs

```diff
@@ -17,6 +17,16 @@ namespace IO.Didomi.SDK
     /// </summary>
     public class Didomi
     {
+        /// <summary>
+        /// Possible views to open in Preferences screen
+        /// </summary>
+        public enum Views
+        {
+            Purposes = 0,
+            // Mapping to Objective-C: Value 1 is SPI (deprecated)
```

Comment:
Should we keep SPI / value 1 and make it deprecated? If not I would keep value 1 since it would lead to the same place and would not require any change when we remove SPI from native.

Reply:
IMO we should not add a value if it is already deprecated. Anyway the native enum will only be removed in a major update, so we will have to remind to update this value (we will have to update whether we add SPI value or not, and unfortunately we won't have an automated warning when native values change)